scaler/entry_points/top.py

```diff
@@ -24,11 +25,11 @@
     ord("f"): "free",
     ord("w"): "sent",
     ord("d"): "queued",
-    ord("s"): "suspended",
+    ord("s"): "n_suspended",
```

Comment:
Shouldn't we use `n_sent` and `n_queued` then?

Reply:
initially I want say n_suspended are for number of suspended processors, but if saying suspended tasks which is also right, change to suspended then, otherwise it will be too many words to fit